app/debug.conf

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file can be used as an overlay to enable debug-related Kconfig options
```

Comment:
The right term would be a Kconfig fragment.
```suggestion
# This is a Kconfig fragment which can be used to enable debug-related options
```

Reply:
changed